app/controllers/admin/stats_controller.rb

```diff
@@ -0,0 +1,70 @@
+# -*- encoding : utf-8 -*-
+
+class Admin::StatsController < ApplicationController
+  def index
+    # total user count
+    @users_count = User.count
+
+    # The most recent users to join
+    @users_stats = params.fetch(:users_stats, {})
+    @recent_users = recent_users
+
+    # Query Solr for top depositors
+    @active_users = top_depositors
+
+    # Count of documents by permissions
+    @files_count = document_by_permission
+
+    # Query Solr for top file formats
+    @top_formats = top_formats
+
+    render 'index'
+  end
+
+  private 
+  def top_depositors_count
```

Comment:
Can you add an extra level of indent for private methods. http://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#follow-the-coding-conventions